episodes/server_resources.md

```diff
@@ -0,0 +1,211 @@
+(episode-server)=
+# Selecting the optimal server resources for your computational work responsibly
+
+teaching: 15
+exercises: 15
+
+:::::{card} 
+OVERVIEW
+^^^
+
+::::{grid}
+:gutter: 2
+
+:::{grid-item}
+Questions
+
+- What's really happening in the cloud when I run a server?
+- How many CPU cores and RAM do I need?
+- How can I reinstate the default environment of a user's server?
+- How can I gain control of user's server for troubleshooting purposes?
+
+:::
+
+:::{grid-item}
+Objectives
+
+- Select the most appropriate server resources for computational work
+- Stop and restart a user's server
+- Support a hub user by gaining control of their server
+
+:::
+
+:::::
+
+## JupyterHub server options
+
+A hub provides interactive computing in the cloud to support research and education workflows for your community. Computational resources, such as the number of {ref}`CPU<glossary:cpu>` cores and {ref}`RAM<glossary:ram>`, can be provisioned on-demand and therefore allows users to scale up or scale down their work as needed.
+
+Once a user is logged into the hub, they can launch a server configured with a number of {ref}`CPU<glossary:cpu>` cores and {ref}`RAM<glossary:ram>`.
```

Comment:
Are the glossary references necessary a second time?

Reply:
Ah yes, that's me being accidentally over-zealous.

```suggestion
Once a user is logged into the hub, they can launch a server configured with a number of CPU cores and RAM`.
```